### DISCUSSION TITLE
MQTT/Alice

### DISCUSSION THREAD

**Discussion by embuiuser** [DmytroKorniienko/FireLamp_EmbUI#113] [General]
Вітаю. Підкажіть, яку команду вводити для керування гучністю програвача? Через МКТ сервер, потрібно для Аліси. Яскравістю вдалося управляти через ***/embui/set/g_bright

**Comment by DmytroKorniienko** [comment]
>Підкажіть, яку команду вводити для керування гучністю програвача?

Зараз немає керування гучністю плеере за допомогою mqtt або http-get. Але це елементарно реалізується, зроблю коли буде натхнення та час.

**Reply by embuiuser** [reply]
Дякую, чекаю. Це єдине, чого мені не вистачає зараз у лампі.

**Comment by embuiuser** [comment]
Можу відредагувати потрібний файл у потрібній папці, скажіть, які рядки внести зроблю. Напевно, це буде найшвидше. Бажаю настрою та натхнення. І часу :)

**Comment by DmytroKorniienko** [comment]
@embuiuser 
Реалізовано в /dev

/embui/get/mp3_volume
/embui/set/mp3_volume

![image](https://user-images.githubusercontent.com/26786760/216645341-58c182c9-9a71-401e-b7ab-c2f69d7f0f40.png)
